# This custom message
int8[]  voxel_coordinate    # Coordinate of the voxel in grid coordinates, 3-tuple
int8    voxel_value         # Value/state of the voxel at aforementioned coordinates